config/peerpods/podvm/osc-podvm-create-job.yaml

```diff
@@ -49,11 +50,45 @@ spec:
             - configMapRef:
                 name: aws-podvm-image-cm
                 optional: true
+            - configMapRef:
+                name: libvirt-podvm-image-cm
+                optional: true
+            - secretRef:
+                name: peer-pods-secret
+                valueFrom:
+                  secretKeyRef:
+                    key: LIBVIRT_URI
+                    name: LIBVIRT_URI
+                optional: true
+            - secretRef:
+                name: peer-pods-secret
+                valueFrom:
+                  secretKeyRef:
+                    key: LIBVIRT_VOL_NAME
+                    name: LIBVIRT_VOL_NAME
+                optional: true
+            - secretRef:
+                name: peer-pods-secret
+                valueFrom:
+                  secretKeyRef:
+                    key: LIBVIRT_POOL
+                    name: LIBVIRT_POOL
+                optional: true
           command: ["/podvm-builder.sh", "create"]
           volumeMounts:
             - name: payload
               mountPath: /payload
+            - mountPath: "/root/.ssh/"
+              name: ssh-key-secret
+              readOnly: true
```

Comment:
Cann't we do -mountpath as optional as it is not used by other cloud providers

Reply:
ok